src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java

```diff
@@ -461,6 +470,32 @@ public EncodingOptions buildOptions() {
 		return args.build();
 	}
 
+	/**
+	 * convert milliseconds to hh:mm:ss.ms representation
+	 * 
+	 * @param value
+	 * @return
+	 */
+	private String millisecondsToString(Long value) {
+		long milliseconds, seconds, minutes, hours = 0;
```

Comment:
`long milliseconds, seconds, minutes, hours = 0;` only sets hours to zero, the rest remind undefined (which defaults to zero). Please set each one to zero, or just move the defintion of seconds/minutes/hours to their first use below. For example `long seconds = milliseconds / 1000;`